docs/datacontract.md

```diff
@@ -432,6 +432,61 @@ reference an external json file that contains the fileDefinition model. The path
     "split_column_name": "observationCodeText",
     "split_value_column_name": "observationValue"
   }
+}
+      </pre>
+    </td>
+  </tr>
+  <tr>
+    <td>validate_value</td>
+    <td>
+    Validates the value of the column against the provided regex to confirm a complete match. An 
+    alternative value can be provided which will be used in case regex does not match
+    </td>
+    <td>
+    <b>column_name:</b> name of column to validate<br>
+    <b>regex:</b> regex to validate against<br>
+    <b>no_match_replacement:</b> replacement value if the regex does not match, defaults to None<br>
+    </td>
+    <td>
+      <pre>
+{
+  "name": "add_constant",
```

Comment:
It may be helpful to change the task name here to "validate_email" or something similar

Reply:
oh my bad.. that task is called `validate_value` because it not just limited to emails.